Splunk_TA_paloalto/lookups/pan_system_change.csv

```diff
@@ -0,0 +1,2 @@
+log_subtype,action,change_type,result,status,command,object,object_category
+general,cleared,filesystem,Config cleared,success,clear,log,file
```

Comment:
We noticed that in the merged code, this lookup is not included. This lookup will populate some of the CIM recommended fields for the Change Data Model. So, if the lookup is not included, the users won't be able to find the mentioned CIM fields for the event.

Reply:
This is tagging only one very specific action that almost never happens in production. So it doesn't really do anything. But similar to the config lookup, I'd like to prioritize specific system messages we're interested in and tag them. Since this would be additive and not a breaking change, we can do it at any time after the 7.0.0 release.